src/sagemaker/image_uri_config/neo-mxnet.json

```diff
@@ -2,20 +2,20 @@
     "processors": ["cpu", "gpu"],
     "scope": ["inference"],
     "version_aliases": {
-        "0.12.1": "1.5",
-        "1.0.0": "1.5",
-        "1.1.0": "1.5",
-        "1.2": "1.5",
-        "1.2.0": "1.5",
-        "1.2.1": "1.5",
-        "1.3": "1.5",
-        "1.3.0": "1.5",
-        "1.4": "1.5",
-        "1.4.0": "1.5",
-        "1.4.1": "1.5"
+        "0.12.1": "1.7",
+        "1.0.0": "1.7",
+        "1.1.0": "1.7",
+        "1.2": "1.7",
+        "1.2.0": "1.7",
+        "1.2.1": "1.7",
+        "1.3": "1.7",
+        "1.3.0": "1.7",
+        "1.4": "1.7",
+        "1.4.0": "1.7",
+        "1.4.1": "1.7"
```

Comment:
Is `1.7` now the only version we provide Neo images for?

Reply:
Neo Service only support one version the latest so older version of the images won't work with the compiled model anyway.